include/irr/asset/IAssetManager.h

```diff
@@ -165,8 +165,13 @@ namespace asset
 			}
 		}
 
+		//TODO change name
         //! _supposedFilename is filename as it was, not touched by loader override with _override->getLoadFilename()
-        //TODO change name
+		/**
+			Attempting of getting an Asset. Valid loaders for certein extension are searched. If none of those can't
```

Comment:
"Attempts to fetch an Asset."

"certein" is a typo -> certain

Reply:
status?

Reply:
Solved